cross-media-measurement/src/main/kotlin/org/wfanet/measurement/common/crypto/SigningFlags.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.wfanet.measurement.common.crypto
+
+import java.io.File
+import picocli.CommandLine
+
+class SigningFlags {
+  @CommandLine.Option(
+    names = ["--signing-certificate"],
+    description = ["Filesystem path to X.509 certificate in PEM format."],
+    required = true
+  )
+  lateinit var certificate: File
+    private set
+
+  @CommandLine.Option(
+    names = ["--signing-private-key"],
+    description = ["Filesystem path to PKCS#8 private key for signing certificate in PEM format."],
+    required = true
+  )
+  lateinit var privateKey: File
+    private set
+
+  @CommandLine.Option(
+    names = ["--trusted-certificate-collection"],
+    description = ["Filesystem path to trusted X.509 certificate collection in PEM format."],
+    required = true
+  )
```

Comment:
This works locally. How are we going to handle this in Kubernetes?

Reply:
The signing certificate and trusted cert collection need not be kept secret and can just be loaded from any volume. The private key can be loaded from a K8s secret used as a volume.

That said, we may want to load these from some kind of keystore instead. Let me go ahead and drop the flags from this PR and address later.